src/main/game/Board.java

```diff
@@ -138,4 +138,13 @@ private boolean gameIsTied() {
         return !playerHasWon(PLAYER_ONE_MARK.getPlayer()) && !playerHasWon(PLAYER_TWO_MARK.getPlayer());
     }
 
+    public List<String> getAvailableMoves() {
+        List<String> availableMoves = new ArrayList<>();
+        for (String space : this.grid) {
+            if (!space.equals(PLAYER_ONE_MARK.getPlayer()) && !space.equals(PLAYER_TWO_MARK.getPlayer())) {
```

Comment:
you could extract that condition out into a function and call it `isSpaceFree` or something like that